### PR TITLE
Images without an alt tag will now give a warning

### PR DESCRIPTION
**Pull request by gja**

### CODE REVIEW

File: components/responsive-image.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ function responsiveProps(props) {
 
 export class ResponsiveImageBase extends React.Component {
   constructor(props, context) {
+    if(process.env.NODE_ENV == 'development' && !props.alt) {
+      console && console.warn(`Image Found without an alt attribute: ${responsiveProps(props).src}`);
```
Comment (thisizkp) [inline components/responsive-image.js]: doubt: why `console` && `console.warn` ?

**Comment by rvgpl** [comment]
@gja  

Confused. ResponsiveImage component doesn't make use of the `alt` prop or I'm missing something?  https://github.com/quintype/quintype-node-components/blob/dd78182edbb96c984f8da249a6f0fc3cb57e13df/components/responsive-image.js#L52

File: components/responsive-image.js (hunk 1 above)
Comment (gja) [reply components/responsive-image.js]: Some browsers don't support console.

**Comment by gja** [comment]
@rvgpl It uses an omit()... So any props that were passed to `ResponsiveImage` are passed directly onwards.

**Pull request merged**